bin/node/src/chain_spec.rs

```diff
@@ -337,21 +297,14 @@ fn configure_distinct_staking_accounts(
                 .into_iter()
                 .map(|account| (account, TOKEN)),
         )
-        .chain(
-            stashes
```

Comment:
shouldn't we endow `authorities` with this `MIN_VALIDATOR_BOND` amount then?

Reply:
We don't need to, since in local runs validator == stash, and validators are already endowed